docs/user/agent/authentication-authorization.md

```diff
@@ -0,0 +1,160 @@
+# Agent-Based Installer Authentication and Authorization - OpenShift 4.18
+
+## Introduction
+Since OpenShift 4.18, the Assisted Service (AS) APIs used by the Agent-Based Installer (ABI) have been secured through authentication and authorization mechanisms. This added security ensures that only authorized services can interact with AS. This document provides an overview of how the authentication and authorization mechanisms implemented for ABI.
```

Comment:
```suggestion
Since OpenShift 4.18, the Assisted Service (AS) APIs used by the Agent-Based Installer (ABI) have been secured through authentication and authorization mechanisms. This added security ensures that only authorized services can interact with AS. This document provides an overview of how the authentication and authorization mechanisms were implemented for ABI.
```